test/tests/functional/pbs_offline_vnodes.py

```diff
@@ -273,9 +273,6 @@ def test_multi_mom_hook_failure_affects_vnode2(self):
         when executed by sister mom, causing
         the server to execute the fail_action=offline_vnodes, which
         result in sister vnode to be marked offline.
-        Then run an exechost_startup hook that gets an
-        exception when local mom is restarted. Vnode representing
-        local mom would be marked offline.

```

Comment:
Indentation appears off by one space in some of the surrounding lines.

Reply:
Hey @mike0042 I ran pycodestyle and I do not see anything. Travis is also not complaining about that.

camac11:functional anamika$ pycodestyle pbs_offline_vnodes.py 
camac11:functional anamika$ echo $?
0

Reply:
I looked at the raw version of the file, and it looks fine. It's just the way GitHub is displaying the diff. Thanks for confirming.